docs/contributing.md

```diff
@@ -76,33 +76,11 @@ If the feature you're interested in is a JavaScript API, you can cross-reference
+Many browsers within BCD can be derived from other browsers given they share the same engine, for example Opera derives from Chrome, and Firefox Android derives from Firefox. To help cut down time working on copying values between browsers, contributors may specify a value in the data to mirror the data for browsers from their upstream counterparts. See the [schema documentation](../schemas/compat-data-schema.md#mirroring-data) for more info.
 
-The <browser> argument is the destination browser that values will be copied to. The script automatically determines what browser to copy from based upon the destination (see table below).
-
-| Destination      | Default Source |
-| ---------------- | -------------- |
-| Chrome Android   | Chrome         |
-| Edge             | Chrome         |
-| Firefox Android  | Firefox        |
-| Opera            | Chrome         |
-| Opera Android    | Chrome Android |
-| Safari iOS       | Safari         |
-| Samsung Internet | Chrome Android |
-| WebView          | Chrome Android |
-
-The <feature_or_filename> argument is either the identifier of the feature to update (i.e. `css.at-rules.namespace`), a filename (`javascript/operators/arithmetic.json`), or an entire folder (`api`).
-
-Note when using feature identifiers:
-
-- Updates aren't applied recursively; only the given data point is updated when using a feature ID. Use a filename or folder for mass mirroring.
-- The script assumes a predictable file structure when passing in a feature identifier, which BCD doesn't have right now. (See [issue 3617](https://github.com/mdn/browser-compat-data/issues/3617).) For example, even if "html.elements.input.input-button" is a valid query, it will fail because the file structure for input-button isn't consistent with the rest right now.
-
-By default, the mirroring script will only overwrite values in the destination that are `true` or `null`, but can take a `--modify` (or `-m`) argument to specify whether to overwrite values that are `false` as well (`--modify=bool`), or any values (`--modify=always`).
-
-The script can also take a `--target_version` (or `-t`) argument to only perform mirroring if the data affects a specific browser release. This is used to update data when a new Opera or Opera Android version is released, and we wish to update data that only affects the new version.
+Note: originally, this functionality used to be provided as an executable script. However, because the script must be run manually, this meant that mirrored data would become stale rapidly. The proposal to move the mirroring into a build step was proposed in [#15083](https://github.com/mdn/browser-compat-data/issues/15083) to reduce maintenance time.
```

Comment:
```suggestion
Note: originally, this functionality used to be provided as an executable script. However, because the script had to be run manually, this meant that mirrored data would become stale rapidly. It was proposed in [#15083](https://github.com/mdn/browser-compat-data/issues/15083) to move the mirroring into a build step to reduce maintenance time.
```